--- ch32l103_it.c.org	2023-06-28 20:39:08.000000000 +0900
+++ ch32l103_it.c	2024-06-17 14:48:04.910869157 +0900
@@ -11,8 +11,8 @@
  *******************************************************************************/
 #include "ch32l103_it.h"
 
-void NMI_Handler(void) __attribute__((interrupt("WCH-Interrupt-fast")));
-void HardFault_Handler(void) __attribute__((interrupt("WCH-Interrupt-fast")));
+void NMI_Handler(void) __attribute__((interrupt("WCH-Interrupt-fast"))) __attribute__(( weak ));
+void HardFault_Handler(void) __attribute__((interrupt("WCH-Interrupt-fast"))) __attribute__(( weak ));
 
 /*********************************************************************
  * @fn      NMI_Handler
