--- ch32v30x_eth.h.org	2022-12-05 16:51:14.000000000 +0900
+++ ch32v30x_eth.h	2024-05-09 12:44:57.934680669 +0900
@@ -955,9 +955,9 @@
 #define ETH_MACA3LR_MACA3L   ((unsigned int)0xFFFFFFFF)  /* MAC address3 low */
 
 /******************************************************************************/
-/*
-/*                          ETH MMC Register
-/*
+/*                                                                            */
+/*                          ETH MMC Register                                  */
+/*                                                                            */
 /******************************************************************************/
 #define ETH_MMCCR_MCFHP      ((unsigned int)0x00000020)  /* MMC counter Full-Half preset */
 #define ETH_MMCCR_MCP        ((unsigned int)0x00000010)  /* MMC counter preset */
@@ -996,9 +996,9 @@
 
 
 /******************************************************************************/
-/*
-/*                          ETH Precise Clock Protocol Register
-/*
+/*                                                                            */
+/*                          ETH Precise Clock Protocol Register               */
+/*                                                                            */
 /******************************************************************************/
 #define ETH_PTPTSCR_TSCNT       ((unsigned int)0x00030000)  /* Time stamp clock node type */
 #define ETH_PTPTSSR_TSSMRME     ((unsigned int)0x00008000)  /* Time stamp snapshot for message relevant to master enable */
@@ -1039,9 +1039,9 @@
 #define ETH_PTPTSSR_TSSO     ((unsigned int)0x00000010)  /* Time stamp seconds overflow */
 
 /******************************************************************************/
-/*
-/*                       ETH DMA Register
-/*
+/*                                                                            */
+/*                       ETH DMA Register                                     */
+/*                                                                            */
 /******************************************************************************/
 #define ETH_DMABMR_AAB       ((unsigned int)0x02000000)  /* Address-Aligned beats */
 #define ETH_DMABMR_FPM        ((unsigned int)0x01000000)  /* 4xPBL mode */
